docs/redis.md

```diff
@@ -49,6 +49,7 @@ The message body should be a JSON object with the following fields:
 
 - `input`: a JSON object with the same keys as the [arguments to the `predict()` function](python.md). Any `File` or `Path` inputs are passed as URLs.
 - `webhook`: the URL Cog will send responses to.
+- `webhook_events` (optional): a list of events to send webhooks for. Must contain `done`, can also contain `start`, `output` and `logs`. Defaults to all events.
```

Comment:
I agree with Ben that

- `webhook_events_filter` might be a more self-documenting name for this, and
- `done` should be `completed` as that's the word we use externally.

I'm also not sure how I feel about requiring that the completed event (whatever we call it) should be in this list, as that seems to me to make it easier to provide invalid input for no obvious benefit. I would suggest that we both:

- always send a `completed` event
- allow `completed` in this list